docs/core/compatibility/5.0.md

```diff
@@ -193,4 +193,5 @@ If you're migrating an app to .NET 5, the breaking changes listed here might aff
 
 ## See also
 
+- [C# compiler breaking changes in C# 9 / .NET 5](https://github.com/dotnet/roslyn/blob/main/docs/compilers/CSharp/Compiler%20Breaking%20Changes%20-%20DotNet%205.md)
```

Comment:
FWIW, it's going to be hard to align the documents perfectly.
If I recall correctly, in the past we did this:
`[.NET 5   document covers this [ .NET 6 `

For example, we used to say something like:
> This document lists known breaking changes in Roslyn in C# 9.0 which will be introduced with .NET 5

But now I think we do this:
`.NET 7  ] document covers this .NET 8 ]`

For example:
> This document lists known breaking changes in Roslyn after .NET 7 all the way to .NET 8.

In the new model, we start a new document right after shipping and we close that document when shipping the next version.

Tagging @cston @333fred to make sure I didn't get confused.

Do you know what model .NET at large uses?

Reply:
Yeah, I did see that this will be a bit trickier.

Adding @gewarren I believe in these docs, the ".NET 8" is all breaking changes in the .NET 8 SDK. So, something changes between .NET 7 (any SDK version) and .NET 8. 

This will also be trickier because some of the compiler breaking changes are observed even with old TFMs. Other than security patches, I don't think that's true for libraries. @gewarren can verify.

Reply:
> Adding @gewarren I believe in these docs, the ".NET 8" is all breaking changes in the .NET 8 SDK. So, something changes between .NET 7 (any SDK version) and .NET 8.

Yes, that's correct.